### PR TITLE
Fix describe with aggregations and parameters

### PR DESCRIPTION
**Pull request by rschlussel-zz**
Fixes aggregation analysis for describe input and output queries with
parameters in the output expressions.

FYI @brian-rickman

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java [hunk 1]
```diff
@@ -89,9 +89,11 @@
     private final Set<Expression> columnReferences;
     private final List<Expression> parameters;
 
+    private final boolean isDescribe;
+
```
Comment (akshatnair) [inline presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java:93]: extra line

**Comment by akshatnair** [comment]
lgtm mod small ccomment

**Comment by akshatnair** [comment]
also check Travis failures

**Comment by rschlussel-zz** [comment]
travis failures are different from each other and both unrelated.  One has a lot of  no worker nodes available failures in product tests.  The other fails because the vm crashed.  I can kick one of them off again to get a passing build.

File: presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java (hunk 1 above)
Comment (rschlussel-zz) [reply presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java:93]: thanks

**Review by petroav — APPROVED** [review]

**Pull request closed without merging**